deployment/run_server

```diff
@@ -1,10 +1,34 @@
 #!/bin/bash -xe
 
 fc-cache -v /usr/share/fonts/user
+set -e
+
+BUILD_DIR=/tmp/plotly-graph-exporter-build
+URL_REGEXP='(https?|ftp|file)://[-A-Za-z0-9\+&@#/%?=~_|!:,.;]*[-A-Za-z0-9\+&@#/%=~_|]'
+ARGS=''
+
+while [[ $# -gt 0 ]]
```

Comment:
Do you need something this complicated? This looks fragile, for example if we need to pass single arguments like `--debug`.

Alternatives:
* Pass the plotly.js URL as an environment variable. If the variable is set, `run_server` does the download and adds the appropriate `--plotlyJS PATH` arguments to another variable (e.g. `PLOTLYJS_ARG`). Then you can run the server as `[...] ./bin/plotly-export-server.js $@ $PLOTLYJS_ARG [...]`.
* Pass the plotly.js URL as the first argument, or `--` if there isn't one. If the argument is not `--`, run_server does the download and adds the path arguments as above.

Reply:
We can fix this by using only one ‘shift’ in the ‘else’ block, but i am also okay with reserving ‘$1’ for plotlyJS argument.. (which can be a url, version-tag or ‘—‘)

Reply:
The one `shift` approach essentially duplicates most of `plotly-export-server.js`'s argument parsing code into `run_server`, which is why I'm not in favour of it.

Is there any reason not to go with the environment variable-based approach? That seems likely to be the most reliable approach since it doesn't involve parsing the argument string in any way.

Reply:
Environment Var based approach sounds okay to me.. !! We can also remove 'plotlyJS' arg from Dockerfile.. !!